src/components/drawer/drawer.js

```diff
@@ -10,12 +11,18 @@ export class DrawerComponent {
   querySelectors() {
     this.drawerRoot = this.mountPoint.querySelector(".mdc-drawer--persistent");
     this.menu = this.mountPoint.querySelector(".toolbar__menu");
+
     this.addAccountButton = this.mountPoint.querySelector(
       ".drawer__add-account-dialog-activation"
     );
     this.addTagButton = this.mountPoint.querySelector(
       ".drawer__add-tag-dialog-activation"
     );
+
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
+      ".drawer__mount-point"
+    );
+
     this.aboutButton = this.mountPoint.querySelector(
```

Comment:
why is it here? it is not your selector, but we don't need this. could you pls remove this.aboutButton from drawer?